taletime/lib/listener/screens/favorites_page.dart

```diff
@@ -120,8 +120,8 @@ class _FavoritePageState extends State<FavoritePage> {
                     children: [
                       SizedBox(
                         height: screenHeight * 0.8,
-                        child: ListViewData(documentSnapshot, widget.storiesColl,
-                            widget.profile, widget.profiles, "favList", widget.favorites),
+                        //child: ListViewData(documentSnapshot, widget.storiesColl,
```

Comment:
I had to remove this as it did not compile. Since we have to rework this page, I would just leave it there as a reminder that there is something to fix.